src/twister2/specification_processor.py

```diff
@@ -252,6 +253,18 @@ def should_skip_for_pytest_harness(test_spec: YamlTestSpecification, platform: P
     return False
 
 
+def should_skip_for_depends_on(test_spec: YamlTestSpecification, platform: PlatformSpecification) -> bool:
+    not_supported_dependencies = []
+    for dependency in test_spec.depends_on:
+        if dependency not in platform.supported:
+            not_supported_dependencies.append(dependency)
+    if not_supported_dependencies:
+        reason = f'"{_join_strings(not_supported_dependencies)}" not occur in platform.supported hardware elements'
```

Comment:
```suggestion
        reason = f'"{_join_strings(not_supported_dependencies)}" not occur in the "supported" section in the platform's definition yaml'
```
This supported list is a mix of keywords, not only hardware, but also some protocols or other functionality. I think such wording will be more accurate

Reply:
Done